cg/services/orders/order_service/models.py

```diff
@@ -0,0 +1,20 @@
+from pydantic import BaseModel, field_validator, Field
+from cg.constants import Workflow
+from cg.server.dto.orders.orders_request import OrderSortField, SortOrder
+
+
+class OrderQueryParams(BaseModel):
+    page_size: int | None = Field(default=50)
+    page: int | None = Field(default=1)
+    sort_field: str | None = Field(default=OrderSortField.ORDER_DATE)
+    sort_order: str | None = Field(default=SortOrder.ASC)
```

Comment:
Should the defaults be removed from the OrderRequest object?

Reply:
I would leave it as is.